docs/reference-architectures/virtual-machines-linux/single-vm.md

```diff
@@ -212,14 +209,17 @@ For more information on deploying this sample reference architecture, visit our
 [disk-encryption]: /azure/security/azure-security-disk-encryption
 [enable-monitoring]: /azure/monitoring-and-diagnostics/insights-how-to-use-diagnostics
 [fqdn]: /azure/virtual-machines/virtual-machines-linux-portal-create-fqdn
+[git]: https://github.com/mspnp/reference-architectures/tree/master/virtual-machines/single-vm
 [github-folder]: https://github.com/mspnp/reference-architectures/tree/master/virtual-machines/single-vm
 [iostat]: https://en.wikipedia.org/wiki/Iostat
 [manage-vm-availability]: /azure/virtual-machines/virtual-machines-linux-manage-availability
+[multi-vm]: multi-vm.md
+[naming-conventions]: /azure/architecture/best-practices/naming-conventions.md
```

Comment:
Use relative path to MD file (see other comment below)

Reply:
Will fix